src/cosmology/compat/camb/__init__.py

```diff
@@ -54,6 +54,11 @@ def hubble_distance(self) -> Array:
         """Hubble distance."""
         return np.array(299792.458 / self.pars.H0)
 
+    @property
+    def critical_density0(self) -> Array:
+        """Critical density today."""
```

Comment:
Can you add a docstring with the definition

Reply:
Done